src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar--month--length-28 calendar--start--day--sat">
```

Comment:
[Checklist #2](https://github.com/mate-academy/layout_calendar/blob/master/checklist.md) - [BEM] - Make sure to follow BEM naming convention for complex modifiers: `block-name--modifier-name--modifier-value`;
```suggestion
      <div class="calendar calendar--month-length--28 calendar--start-day--sat">
```

Reply:
In this case
modifier-name = month-length
modifier-value = 28

Reply:
Don't forget to update scss too